includes/platform-checkout/class-platform-checkout-utilities.php

```diff
@@ -39,14 +39,14 @@ public function should_enable_platform_checkout( $gateway ) {
 	/**
 	 * Checks various conditions to determine if WooPay should be enabled on the checkout page.
 	 *
-	 * This function should only be called when evaluating something for the checkout page. The
+	 * This function should only be called when evaluating something for the checkout or cart page. The
 	 * function will return false if you're on any other page.
 	 *
 	 * @return bool  True if WooPay should be enabled, false otherwise.
 	 */
 	public function should_enable_woopay_on_checkout(): bool {
```

Comment:
Let's rename the function to `should_enable_woopay_on_cart_or_checkout()` the function is only called in one place so it should be an easy change.

Reply:
Great catch, thanks! Fixed.